### PR TITLE
PM-15067 - Design Audit - Prevent Account Lockout Screen

### PR DESCRIPTION
**Pull request by phil-livefront**
## 🎟️ Tracking

[PM-15067](https://bitwarden.atlassian.net/browse/PM-15067)

## 📔 Objective

- Center align the title and subtitle text
- Updated the `PreventAccountLockoutScreen` to use the `BitwardenContentCard` which matches the figma designs

## 📸 Screenshots

PreventAccountLockoutScreen

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-15067]: https://bitwarden.atlassian.net/browse/PM-15067?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/preventaccountlockout/PreventAccountLockoutScreen.kt [hunk 1]
```diff
@@ -75,102 +68,57 @@ fun PreventAccountLockoutScreen(
             )
         },
     ) {
-        Column(
+        NeverLoseAccessContent(
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/preventaccountlockout/PreventAccountLockoutScreen.kt:71]: Can we call this `PreventAccountLockoutContent` to keep it consistent

File: app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt [hunk 2]
```diff
@@ -107,7 +108,7 @@ private fun BitwardenContentBlock(
                 ?: Spacer(Modifier.width(16.dp))
         }
 
-        Column {
+        Column(modifier = Modifier.padding(end = 16.dp)) {
```
Comment (david-livefront) [inline app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt:111]: Can we leave this in the spacer?

Comment (phil-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt:111]: I couldn't get it to work as it was, it was still going to the very edge of the view.  Any ideas why? On iOS we use spacers to space out views easily vs adding a spacer for padding reasons so I was pretty confused here haha.

Comment (david-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt:111]: Wanna give this a whirl:

```kotlin
        Column(modifier = Modifier.weight(weight = 1f, fill = false)) {
            Spacer(Modifier.height(12.dp))
            Text(
                text = headerText,
                style = headerTextStyle,
            )
            subtitleText?.let {
                Spacer(Modifier.height(4.dp))
                Text(
                    text = it,
                    style = subtitleTextStyle,
                    color = BitwardenTheme.colorScheme.text.secondary,
                )
            }
            Spacer(Modifier.height(12.dp))
        }
        Spacer(Modifier.width(16.dp))
```

Comment (phil-livefront) [reply app/src/main/java/com/x8bit/bitwarden/ui/platform/components/content/BitwardenContentBlock.kt]: worked like a charm.  thanks!

**Review by david-livefront — APPROVED** [review]

**Review by dseverns-livefront — APPROVED** [review]

**Pull request merged**